vm/systemSmartContracts/delegation.go

```diff
@@ -810,6 +814,10 @@ func (d *delegation) setAutomaticActivation(args *vmcommon.ContractCallInput) vm
 
 	switch string(args.Arguments[0]) {
 	case "true":
+		if d.enableEpochsHandler.IsFlagEnabled(common.AutomaticActivationOfNodesDisableFlag) {
```

Comment:
so we return user error if the transaction is with automatic activation but the automatic activation is no longer supported by the protocol, enforcing the user to explicitly set it.

I think it would be better to simply ignore the argument value without returning an error in this case.